build.sh

```diff
@@ -208,6 +208,19 @@ isMSBuildOnNETCoreSupported()
         elif [ "$__BuildOS" == "OSX" ]; then
             __isMSBuildOnNETCoreSupported=1
         fi
+    elif [ "$__BuildArch" == "arm" ] || [ "$__BuildArch" == "arm64" ] ; then
+        if [ "$__BuildOS" == "Linux" ]; then
+            if [ "$__DistroName" == "ubuntu" ]; then
+                __isMSBuildOnNETCoreSupported=1
```

Comment:
@gkhanna79, @benpye, I agree. I will submit next PR based on a merged latest commit no. to keep only the Distro/OS that we have been tested.